django-verdant/rca/templates/rca/blocks/video_block.html

```diff
@@ -1,7 +1,18 @@
 {% load wagtailembeds_tags wagtailimages_tags %}
 
-{{ value.title }}
+<div class="videoembed vimeo">
+    <h2 class="block-video__title homepage-h2">{{ value.title }}</h2>
 
-{{ value.video.url|embed }}
+    <button class="playpause play block-video__play-button">
+        <svg class="block-video__play-icon">
```

Comment:
Double check accessibility - does it need a title?

Reply:
added